tests/test_consent/api/test_permissions.py

```diff
@@ -37,6 +38,10 @@ def setUp(self):
         self.create_user(username=TEST_USERNAME, password=TEST_PASSWORD, id=TEST_USER_ID)
         self.client = APIClient()
         self.client.login(username=TEST_USERNAME, password=TEST_PASSWORD)
+        client_class = mock.patch('enterprise.models.CourseCatalogApiClient')
```

Comment:
@haikuginger I'm fine with `cd_client` above (although it might be slightly confusing to someone not familiar with the Course Catalog API/Course Discovery terminology); but can we be a bit more specific here and call this `discovery_client_class` or `catalog_client_class`?